src/utils/jsonToJdl.js

```diff
@@ -168,6 +172,7 @@ application {
         ${clientFramework ? `clientFramework ${applications[i].clientFramework.toLowerCase()}` : 'clientFramework no'}
         ${appFramework ? `blueprint [${applications[i].applicationFramework.toLowerCase()}]` : ''}
         ${withExample ? `withExample true` : ''}
+        ${buildTool !== false ? `buildTool ${applications[i]?.buildTool.toLowerCase()}` : ''}
```

Comment:
condition could be simply `buildTool ?` like above checks

Reply:
Resolved this issue!!